math/lcm.go

```diff
@@ -0,0 +1,49 @@
+// Formula from https://en.wikipedia.org/wiki/Least_common_multiple
+// Find the least common multiple of two integers
+// by using the greatest common divisor.
+
+package main
```

Comment:
When writing a new package, do not leave an empty line after a comment explaining something - this cause issues with automated documentation generation.

Furthermore, in this repository we do not accept packages with name `main` anymore. Please change the package name to something else and remove the `main` function as well.

```suggestion
package lcm
```